certification/artifacts/artifacts.go

```diff
@@ -9,6 +10,16 @@ import (
 	"github.com/spf13/viper"
 )
 
+func WriteFile(filename, contents string) (string, error) {
+	fullFilePath := filepath.Join(Path(), filename)
+
+	err := ioutil.WriteFile(fullFilePath, []byte(contents), 0644)
```

Comment:
`ioutil.WriteFile` is [deprecated/moved](https://golang.org/doc/go1.16#ioutil) in Go 1.16.  Use `os.WriteFile`.

Reply:
Thanks. I'll fix that.